src/content/docs/nf-core-tools/pipelines/create.md

```diff
@@ -9,52 +9,74 @@ The `create` subcommand makes a new pipeline using the nf-core base template.
 With a given pipeline name, description and author, it makes a starter pipeline which follows nf-core best practices.
 
 After creating the files, the command initialises the folder as a git repository and makes an initial commit.
+It also allows the creation of a GitHub repository where this commit is pushed.
 This first "vanilla" commit which is identical to the output from the templating tool is important, as it allows us to keep your pipeline in sync with the base template in the future.
 See the [nf-core syncing docs](https://nf-co.re/developers/sync) for more information.
 
 <!-- RICH-CODEX
 working_dir: tmp
 -->
 
-![` nf-core create -n nextbigthing -d "This pipeline analyses data from the next big omics technique" -a "Big Steve" --plain`](/images/tools/nf-core-create.svg)
+![` nf-core pipelines create -n nextbigthing -d "This pipeline analyses data from the next big omics technique" -a "Big Steve"`](/images/tools/nf-core-create.svg)
 
-Once you have run the command, create a new empty repository on GitHub under your username (not the `nf-core` organisation, yet) and push the commits from your computer using the example commands in the above log.
-You can then continue to edit, commit and push normally as you build your pipeline.
 
 Please see the [nf-core documentation](/docs/tutorials/adding_a_pipeline/overview) for a full walkthrough of how to create a new nf-core workflow.
 
 > [!TIP]
-> As the log output says, remember to come and discuss your idea for a pipeline as early as possible!
+> As command documentation says, remember to come and discuss your idea for a pipeline as early as possible!
 > See the [documentation](/docs/tutorials/adding_a_pipeline/move_to_nf-core_org) for instructions.
 
-Note that if the required arguments for `nf-core create` are not given, it will interactively prompt for them. If you prefer, you can supply them as command line arguments. See `nf-core create --help` for more information.
+Note that if the required arguments for `nf-core pipelines create` are not given, it will start a user interface to create the pipeline interactively. If you prefer, you can supply them as command line arguments. See `nf-core pipelines create --help` for more information.
 
 ## Customization
 
-The `nf-core create` command comes with a number of options that allow you to customize the creation of a pipeline if you intend to not publish it as an
-nf-core pipeline. This can be done in two ways: by using interactive prompts, or by supplying a `template.yml` file using the `--template-yaml <file>` option.
+The `nf-core pipelines create` command comes with a number of options that allow you to customize the creation of a pipeline if you intend to not publish it as an
+nf-core pipeline. This can be done in two ways: by using the graphic interface, or by supplying a `template.yml` file using the `--template-yaml <file>` option.
```

Comment:
```suggestion
nf-core pipeline. This can be done in two ways: by using the graphical interface, or by supplying a `template.yml` file using the `--template-yaml <file>` option.
```